components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -503,6 +503,13 @@ class BrowserToolbar @JvmOverloads constructor(
         editToolbar.addEditAction(action)
     }
 
+    /**
+     * Focuses the editToolbar if already in edit mode
```

Comment:
nit: `editToolbar` is an implementation detail and not visible from the "outside". We should describe what the method does without mentioning it. :)